test-data/unit/check-classes.test

```diff
@@ -8514,12 +8514,23 @@ class C(B):
     @deco_untyped
     def baz(self, x: int) -> None: ...
 
+    @property
+    def tricky(self) -> int: ...
+    @baz.setter
```

Comment:
It feels like this should probably be `@tricky.setter`. Unless this is intentional, but it definitely looks like a copy/paste issue.

Reply:
Looks that way. Thanks for letting us know! Opened https://github.com/python/mypy/pull/18946 to fix it.